lib/sink.js

```diff
@@ -8,52 +8,57 @@ const path = require('path');
 const fs = require('fs');
 
 
-const SinkFs = module.exports = function (fileDir) {
-    if (!(this instanceof SinkFs)) { return new SinkFs(fileDir); }
-    assert(fileDir, '"fileDir" must be provided');
+module.exports = class SinkFs {
+    constructor (fileDir) {
+        assert(fileDir, '"fileDir" is missing');
```

Comment:
Does absolute / relative really matter? 

I think we should throw if path does not exist.